components/train.yaml

```diff
@@ -106,6 +108,22 @@ implementation:
           ${bootleg_args} \
           $@
 
+      if test "$calibrate" == "true" ; then
+        genienlp predict \
+            --path "$modeldir" \
+            --data "dataset" \
+            --eval_dir "eval_dir" \
+            --task ${task_name} \
+            --silent \
+            --overwrite \
+            --evaluate valid \
+            --save_confidence_features \
+            --confidence_feature_path "$modeldir/confidence_feature_file.pkl" \
+            ${calibration_additional_args}
+
+        genienlp calibrate --save "$modeldir" --confidence_path "$modeldir/confidence_feature_file.pkl" --plot
+      fi
```

Comment:
You should also add the code to display the plots. It should just be a matter of creating an HTML visualization object that contains the plot SVG, and writing the visualization JSON to the right file.